ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/mappers/ClaimMapper.java

```diff
@@ -67,7 +67,8 @@ public Claim map(int index, ResultSet result, StatementContext ctx) throws SQLEx
             toNullableClaimState(result.getString("state")),
             toNullableEntity(result.getString("submission_operation_indicators"),
                 ClaimSubmissionOperationIndicators.class),
-            null
+            null,
+            null // Not implemented to save review order in claim store database
```

Comment:
shoulde we delete the migration stuff now?

Reply:
Yes, we have a separate story to clean up this.